### DISCUSSION TITLE
posts/React/React%20useState%20%ED%9B%85%EA%B3%BC%20%ED%81%B4%EB%A1%9C%EC%A0%80

### DISCUSSION THREAD

**Discussion by giscus** [phnml1/my-blog#13] [General]
# posts/React/React%20useState%20%ED%9B%85%EA%B3%BC%20%ED%81%B4%EB%A1%9C%EC%A0%80

클로저를 이용한 React useState 훅의 작동 원리를 이해해보자

https://phnml1.github.io/posts/React/React%20useState%20%ED%9B%85%EA%B3%BC%20%ED%81%B4%EB%A1%9C%EC%A0%80

**Comment by phnml1** [comment]
rrrrr
